src/renderer/features/proxies/model/proxies-model.ts

```diff
@@ -75,14 +77,26 @@ type GetProxiesResult = {
   };
 };
 const getProxiesFx = createEffect(
-  ({ chainId, accounts, proxies, endpoint }: GetProxiesParams): Promise<GetProxiesResult> => {
+  ({ chainId, accounts, wallets, proxies, endpoint }: GetProxiesParams): Promise<GetProxiesResult> => {
     const proxiedAccounts = accounts.filter((a) => accountUtils.isProxiedAccount(a));
-    const nonProxiedAccounts = keyBy(
-      accounts.filter((a) => !accountUtils.isProxiedAccount(a)),
+
+    const accountsForProxy = keyBy(
+      accounts.filter((a) => proxiesUtils.isProxyAvailable(a)),
+      'accountId',
+    );
+    const accountsForProxied = keyBy(
+      accounts.filter((a) => proxiesUtils.isProxiedAvailable(a, wallets[a.walletId])),
```

Comment:
Not sure that we require `walletsMap` because `getProxiesFx` can conveniently use it
It's ok to pass wallets array, build map from it and use just right here

Reply:
done, but we already calculate it in several places, and it may be useful to have it just in one place